steering_controllers/src/steering_controllers.yaml

```
@@ -0,0 +1,164 @@
+steering_controllers:
+  reference_timeout: {
+    type: double,
+    default_value: 1,
+    description: "Timeout for controller references after which they will be reset. This is especially useful for controllers that can cause unwanted and dangerous behaviour if reference is not reset, e.g., velocity controllers. If value is 0 the reference is reset after each run.",
+#    validation: {
+#      gt_eq<>: 0.0,
+#    }
+  }
+
+  rear_wheel_names: {
+    type: string_array,
+    default_value: ["rear_wheel_joint"],
+    description: "Name of rear wheels.",
+    read_only: true,
+
+  }
+
+  front_steer_names: {
```

If we enable both, front and rear steering.

```suggestion
  front_wheels_names: {
```

---

We need then also two additional parameters:

- `rear_steer_joints`
- `front_steer_joints`

In this way, we calculated always steer angle and velocity depending on the configuration. Or would this complicate the code too much?